frontend/pages/post/[id].js

```diff
@@ -305,6 +430,7 @@ class LongPost extends React.Component {
               directHandleChange={this.directHandleChange}
               ClassicEditor={this.ClassicEditor}
             ></PostArticleEdittable>
+            <div>{state}</div>
```

Comment:
I am curious as to what this is?

Reply:
That shouldn't be there, cheers I'll get rid of it.